packages/big-design/src/components/PillTabs/spec.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
```

Comment:
I assume this is the same issue as above, we might be able to get rid of this one with the same approach